js/apps/admin-ui/src/clients/ClientsSection.tsx

```diff
@@ -71,6 +73,15 @@ const ClientDetailLink = (client: ClientRepresentation) => {
           </Badge>
         )}
       </Link>
+      {client.attributes?.["temporary_admin"] == "true" && (
```

Comment:
```suggestion
      {client.attributes?.["temporary_admin"] === "true" && (
```
It's almost always better to use `===` instead of `==`.

Reply:
@ssilvert initially I tried the === but it didn't work properly. I haven't investigated why but I can certainly try it again.

Reply:
@ssilvert just to be more clear, the `===` operator worked for clients, but not for the users. Upon further investigation I guess it's because the clients' attributes are stored in the Map<String, String> data structure while users' attributes are stored in Map<String, List<String>>. 
So given the current state of proposal, in the first case the  type of string is returned (hence the operator works) and in the second case the type of object (`['true']` array) is returned.

So either I can either convert it before the comparison:
`String(user.attributes?.["temporary_admin"]) === "true"` 
or I can compare the value in stored in the object:
`user.attributes?.["temporary_admin"][0] === "true"`

Both works and uses the suggested operator. I proceeded with the second way of doing it as I consider it a bit cleaner but let me know in case of any objections.

cc: @vmuzikar